src/EnergyPlus/EconomicLifeCycleCost.hh

```diff
@@ -96,26 +109,155 @@ namespace EconomicLifeCycleCost {
+                                                                                                    "TOTAL OPERATIONAL",
+                                                                                                    "CONSTRUCTION",
+                                                                                                    "SALVAGE",
+                                                                                                    "OTHER CAPITAL",
+                                                                                                    "TOTAL CAPITAL",
+                                                                                                    "TOTAL ENERGY",
+                                                                                                    "GRAND TOTAL"};
+    constexpr std::array<std::string_view, static_cast<int>(CostCategory::Num)> CostCategoryNamesUCNoSpace{"MAINTENANCE",
+                                                                                                           "REPAIR",
+                                                                                                           "OPERATION",
+                                                                                                           "REPLACEMENT",
+                                                                                                           "MINOROVERHAUL",
+                                                                                                           "MAJOROVERHAUL",
+                                                                                                           "OTHEROPERATIONAL",
+                                                                                                           "WATER",
+                                                                                                           "ENERGY",
+                                                                                                           "TOTALOPERATIONAL",
+                                                                                                           "CONSTRUCTION",
+                                                                                                           "SALVAGE",
+                                                                                                           "OTHERCAPITAL", // No space
+                                                                                                           "TOTALCAPITAL",
+                                                                                                           "TOTALENERGY",
+                                                                                                           "GRANDTOTAL"};
+
+    constexpr std::string_view Total{"Total"};
+    constexpr std::string_view TotalUC{"TOTAL"};
```

Comment:
Is there a better way to do this?

Reply:
Maybe, but I'm not sure if it is worth it or not.  C++ has a `toupper` function that operates on individual characters, but not on a string.  So you'd have to iterate over the characters, and you'd need to construct a string while doing it.  Maybe @lefticus has a fancy idea for how to do this most efficiently, but unless this is a widespread issue on this PR, I'd just put a pin in it.  Getting it into `constexpr string_view` is already such an improvement over the past...

Reply:
This is the fastest and cheapest way to do this unless the `MakeUpper` function can itself be made `constexpr`. I don't think it can, but it may be worth a quick test. Even then the implementation wouldn't be cheaper, it would have the same cost but maybe the code would be a bit clearer. Either way, it's not worth holding up this PR.

Reply:
Thank you for the inputs and insights. 
No, it's not a widespread issue, just felt that it felt a bit weird pulling these out as individual variables. 

Will investigate changing `MakeUpper` into a `constexpr` function separately.

Reply:
https://godbolt.org/z/nKcEf5eev  I've given it 10 minutes and haven't figured out a path forward on it.  I assume that means it is not possible.  :laughing: :rofl: JK, but for real that's a starting point if anyone wants to try to make a clean solution.

Reply:
Where did you learn how to program?

Reply:
More seriously, `makeUpperCase` cannot be made `constexpr` because it has to allocate memory for the target string so this path is closed. Just leave as is for now.